src/main/java/net/minecraftforge/common/ForgeConfigSpec.java

```diff
@@ -811,11 +943,12 @@ public Object getDefault()
 
     public static class ConfigValue<T> implements Supplier<T>
     {
-        private static boolean USE_CACHES = true;
+        private static final boolean USE_CACHES = true;
 
         private final Builder parent;
         private final List<String> path;
         private final Supplier<T> defaultSupplier;
+        private Supplier<ConfigGuiWidgetFactory> screenWidgetFactorySupplier = () -> null;
```

Comment:
Same comment as on previous set of comments, might as well make the `() -> null` supplier that gets fallen back to just have a single instance.